db/seeds.rb

```diff
@@ -19,3 +19,25 @@
 company = Company.first
 
 FactoryBot.create_list(:page, 20, company: company)
+
+200.times { Company.create({
```

Comment:
this is definitely not good..
use factories
als check `::create_list` from FactoryBot